acceptance/tests/vault/vault_wan_fed_test.go

```diff
@@ -229,7 +230,6 @@ func TestVault_WANFederationViaGateways(t *testing.T) {
 		"server.extraVolumes[0].type": "secret",
 		"server.extraVolumes[0].name": vaultCASecretName,
 		"server.extraVolumes[0].load": "false",
-		"server.extraConfig":          serverExtraConfig,
```

Comment:
I was wondering if it would be worthwhile to keep a test where this is set via extra config, but I don't really think it is because at that point it would just be testing something that should be covered by the `bats` tests. So I think it's the right move to just modify the test.

Reply:
We do use server extra config in the mesh_gateway_test which doesn't use vault:

https://github.com/hashicorp/consul-k8s/blob/ae206c9009a1860391b3cb55dfaeea394654234c/acceptance/tests/mesh-gateway/mesh_gateway_test.go#L76-L80